app/src/main/java/nyc/c4q/jonathancolon/inContaq/contactlist/graphs/MonthlyGraph.java

```diff
@@ -188,6 +185,8 @@ synchronized private void loadGraph() {
         animateGraph();
     }
 
+    LineSet dataSetMonthly;
+    LineSet dataset;
```

Comment:
no need to make these fields, they don't seem to be used outside of `setGraphData()`